receiver/huaweicloudcesreceiver/internal/ces_client.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package internal // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/huaweicloudcesreceiver/internal"
+
+import (
+	ces "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/ces/v1"
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/services/ces/v1/model"
+)
+
+//go:generate mockery --name CesClient --case=underscore --output=../mocks
+type CesClient interface {
```

Comment:
we might need to add a comment on how to maintain this interface. it's not auto generated, is it?

Reply:
It is not auto-generated. but in theory we should "never" change it. 
These are the function defined inside https://github.com/huaweicloud/huaweicloud-sdk-go-v3/blob/v0.1.110/services/ces/v1/ces_client.go

So as long as we keep the same version (v0.1.110), no change is required.
Added a comment in the code as well.